spyder/utils/misc.py

```diff
@@ -323,3 +326,41 @@ def check_connection_port(address, port):
         return False
     finally:
         s.close()
+
+
+def get_pyenv_path(name):
+    """Return the complete path of the pyenv."""
+    home = os.environ['USERPROFILE'] if WINDOWS else os.environ['HOME']

```

Comment:
We already have a function to get the user's home directory. It's called `get_home_dir` and it's present in `spyder/config/base.py`